tgg/seqr_scripts/sex_check.py

```diff
@@ -0,0 +1,208 @@
+import argparse
+import logging
+import matplotlib.pyplot as plt
+import pandas as pd
+from os.path import dirname
+
+import hail as hl
+import hail.expr.aggregators as agg
+
+from gnomad.utils.reference_genome import get_reference_genome
+
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s", 
+    datefmt="%m/%d/%Y %I:%M:%S %p", 
+)
+logger = logging.getLogger("sex_check")
+logger.setLevel(logging.INFO) 
+
+
+def get_chr_cov(mt: hl.MatrixTable, build: str, call_rate_threshold: float=0.25, chr_name: str, af_field: str="AF", af_threshold: float=0.01) -> hl.Table:
+    """
+    Calculate mean chromosome coverage. 
+
+    :param mt: MatrixTable containing samples with chrY variants
+    :param build: Reference used, either GRCh37 or GRCh38
+    :param call_rate_threshold: Minimum required call rate
+    :param chr_name: chosen chromosome, either 1, 2, 3, ..., X, or Y
+    :param af_field: allele frequency field of matrix table 
+    :param af_threshold: allele frequency threshold 
+    :return: Table with coverage annotations
+    """
+
+    logger.warning("This function expects the chrY to be at index 23 and chrX to be at index 22.")
+    
+    if chr_name == "Y": 
+        filter_nonpar_expr = sex_mt.locus.in_y_nonpar() 
+        chr_place = 23 
+    elif chr_name == "X": 
+        filter_nonpar_expr = sex_mt.locus.in_x_nonpar()
+        chr_place = 22
+    else: 
+        try:
+            # Chromosome index in '.contigs' list should be one less than the chromosome number 
+            chr_place = int(chr_name) - 1 
+        except ValueError: 
+            logger.warning("chr_name cannot be converted to an integer")
```

Comment:
```suggestion
            logger.error("chr_name cannot be converted to an integer")
            return
```
small change, this should be an error here, and the function should probably exit (or else it will throw an error trying to run line 49 below)